php-practice/section5.php

```diff
@@ -158,7 +168,15 @@ function printLines(array $lines) {
  * So if $min is 5 and $max is 10, it would print out: 5 6 7 8 9 10
  */
 
+    function getValuesBetween() {
+      $max = 10;
 
+      for ($min= 5; $min <= $max; $min++) {
+        echo $min . PHP_EOL;
+      }
+}
+
+echo getValuesBetween();
```

Comment:
Okay, so when I say "takes 2 numbers" I mean I expect the function to have 2 parameters.

Any time you hear, "this function **takes** blah blah blah", it's referring to the parameters that can be passed in.

So I'm looking for this:

```
function getValuesBetween($min, $max) {
  // ...
}
```